misk-zookeeper/src/test/kotlin/misk/zookeeper/ZkTestModule.kt

```diff
@@ -22,12 +22,13 @@ class ZkTestModule(
   override fun configure() {
     val keystorePath = this::class.java.getResource("/zookeeper/keystore.jks").path
     val truststorePath = this::class.java.getResource("/zookeeper/truststore.jks").path
-
-    install(ZookeeperModule(ZookeeperConfig(
+    val config = ZookeeperConfig(
         zk_connect = "127.0.0.1:$zkPortKey",
         cert_store = CertStoreConfig(keystorePath, "changeit", SslLoader.FORMAT_JKS),
-        trust_store = TrustStoreConfig(truststorePath, "changeit", SslLoader.FORMAT_JKS)),
-        qualifier))
+        trust_store = TrustStoreConfig(truststorePath, "changeit", SslLoader.FORMAT_JKS))
+
+    install(ZookeeperCommonModule(config, qualifier))
```

Comment:
it seems like these 2 modules should be packaged up into `ZookeeperModule` for you.